### PR TITLE
EndsWithSingleListApp & EndsWithDualListApp are enable by IsStroustrupStyle as well.

### PR DESCRIPTION
**Pull request by nojaf**
As discussed with @josh-degraw.

### CODE REVIEW

File: src/Fantomas.Core.Tests/Stroustrup/ExperimentalElmish.fs
Comment (josh-degraw) [inline src/Fantomas.Core.Tests/Stroustrup/ExperimentalElmish.fs:1]: Should we not name this file `ExperimentalElmishTests.fs`?

File: src/Fantomas.Core/FormatConfig.fs [hunk 1]
```diff
@@ -227,7 +227,7 @@ type FormatConfig =
 
       [<Category("Convention")>]
       [<DisplayName("Applies the Stroustrup style to the final (two) array or list argument(s) in a function application")>]
```
Comment (josh-degraw) [inline src/Fantomas.Core/FormatConfig.fs:229]: Not sure either way, but should we change anything about this displayName to mention elmish or do you think it's fine to stay as is??

Comment (nojaf) [reply src/Fantomas.Core/FormatConfig.fs:229]: I think this is fine for now. We can revisit this later.
Little secret, but somebody is actually doing anything with these attributes anyway.
We expose them via `Fantomas.Client` so IDEs can request them but none of them has implemented this.

File: src/Fantomas.Core.Tests/Stroustrup/ExperimentalElmish.fs
Comment (nojaf) [reply src/Fantomas.Core.Tests/Stroustrup/ExperimentalElmish.fs:1]: Good catch

**Review by josh-degraw — APPROVED** [review]

**Pull request merged**